scholarships.html

```diff
@@ -2,7 +2,6 @@
 layout: subpage
 page_id: scholarships
 title: "Apply for a scholarship"
-subtitle: "Applications open soon"
```

Comment:
Maybe change this to:
"Applications open until January 20" (or whatever date we decide)